app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= render 'includes/second_navigation', back_link: 'javascript:history.back()', title: 'Add Category' %>
+
+<section>
+  <div class="container h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col-lg-12 col-xl-11">
+        <div class="card text-black" style="border-radius: 25px;">
+          <div class="card-body p-md-5">
+            <div class="row justify-content-center">
+              <div class="col-md-10 col-lg-6 col-xl-5 order-2 order-lg-1">
+
+                <p class="text-center h1 fw-bold mb-5 mx-1 mx-md-4 mt-4">Record a new transaction</p>

```

Comment:
- [ ] Kindly add form validation to your form such that the users have to fill every field before submitting the data.